### PR TITLE
wiki(entombsday): add initial chapters

### PR DESCRIPTION
**Pull request by drynne0**

### CODE REVIEW

File: markdown/chapters/entombsday/deck-construction.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Table of Contents
+
+## Preamble
+
+As previously stated, the shell of both Tin Fins and Doomsday are remarkably similar and the combo packages are roughly the same size. Because of this overlap, there are very few concessions made in deckbuilding.
+
+::decklist{path=2023/03/entombsday.drynne}
+::decklist{path=2023/03/entombsday.Nitro_Rick}
+
+### Notable Inclusions
```
Comment (Doishy) [inline markdown/chapters/entombsday/deck-construction.md:16]: For this section overall I think you need to talk more about the components you need and what each set of tools does. Then discuss the decision making. 

Here we have a list of cards some of which have some useful info, others less so.

File: markdown/chapters/entombsday/interaction.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Cards that Empty the Graveyard
+
+:::row{variant=CENTERED}
+- Bojuka Bog
+- Endurance
+- Faerie Macabre
+:::
+
+Cards like Bojuka Bog, Endurance, and Faerie Macabre are some of the easiest to play around for Tin Fins. The general play pattern to beat these effects is to *bin* your Griselbrand, bait the interaction with a Shallow Grave, the simply cast a second copy of Entomb (or crack LED) to bin another Griselbrand after the effect resolves.
```
Comment (Doishy) [inline markdown/chapters/entombsday/interaction.md:33]: No discussion on patience such as bin, ptt, they go for the interact and you respond with Shallow/Veng? 

I feel you could deffo add more here.

File: markdown/chapters/entombsday/interaction.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Note that Shallow Grave is required here, as it does not target a specific card.
+
+## Surgical Extraction
+
+:::row{variant=CENTERED}
+- Surgical Extraction
+- Extirpate
+:::
+
+While similar to the previously mentioned effects, Surgical Extraction requires the inverse play pattern. Since Surgical Extraction and Extirpate exile all copies of the chosen card, you can't allow it to resolve. To beat these effects, you will need two reanimation effects instead of two Entomb effects.
```
Comment (Doishy) [inline markdown/chapters/entombsday/interaction.md:44]: Extirpate 

Not like you can really play around this one? Split Second kind of hard to hit against.

File: markdown/chapters/entombsday/interaction.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+:::row{variant=CENTERED}
+- StP
+- Solitude
+- Karakas
+:::
+
+Tin Fins is able to play around removal on Griselbrand fairly easily. Most players will wait until you activate Griselbrand the first time before casting a removal spell. Because of this, you can usually activate Griselbrand a second time to find a piece of countermagic to answer the removal spell. 
+
+If you're unable to counter the removal, such as Karakas, it is still correct to draw as many cards as possible before it resolves. Generally, drawing 14 cards is enough to allow you to combo again in the same turn.
```
Comment (Doishy) [inline markdown/chapters/entombsday/interaction.md:56]: Maybe talk about what cards you are looking to find from activations and how to know when to stop and let it happen?

File: markdown/chapters/entombsday/playing-the-deck.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+## Preamble
+
+As a deck with two unique gameplans, Entombsday requires some flexibility from its pilot. Knowing the ins-and-outs of both Tin Fins and Doomsday will improve your performance with the deck, as will developing your own heuristics for when to uses which plan.
+
+Because this Wiki already contains excellent resources for how to play Turbo Doomsday, we will primarily focus on the Tin Fins plan and when to *transform* into Doomsday.
+
+## Playing Tin Fins
+
+Fundamentally, Tin Fins is an A + B reanimator combo deck.
```
Comment (Doishy) [inline markdown/chapters/entombsday/playing-the-deck.md:15]: Surely you can add more than this here?

Is A+B. 
Looking to use cards like Ent to drop GrizzyB into gy, then rean with Haste with SG or GV. You can also use cards like X or Y to loop your deck or just play a slower gameplan reanimating and hitting over consecutive turns.,...

File: markdown/chapters/entombsday/playing-the-deck.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+
+### Infinite Loops
+
+In the event that you can’t find a way to reanimate Emrakul pre-combat, you are able to *loop* your deck to either cast Emrakul or repeatedly cast :card[Collective Brutality] to drain your opponent out 2 life at a time.
+
+To *loop* your deck:
+
+1. Draw 14 to 21 cards with Griselbrand until you can put :card[Children of Korlis] into play (either by casting or reanimating it).
+2. Sacrifice Children of Korlis to regain all of the life you’ve lost so far this turn (this effect is **cumulative**, so you can gain functionally infinite life if you *loop* Children).
+3. Using the life gain from Children, draw your entire library and begin casting Dark Rituals to generate a large amount of {B} mana.
```
Comment (Doishy) [inline markdown/chapters/entombsday/playing-the-deck.md:33]: Be careful not to draw yourself to death.....
Maybe make a note of what do if not find children with 6 cards left etc (use LP and Pn to find them).

File: markdown/chapters/entombsday/basics.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+- Entomb
+- Griselbrand
+- Doomsday
+:::
+
+Tin Fins is a Reanimator variant that primarily aims to reanimate :card[Griselbrand] with haste using either :card[Shallow Grave] or :card[Goryo’s Vengeance], then draw enough cards to reanimate :card[Emrakul, the Aeons Torn] and attack for 22 damage. To enable this, the deck runs :card[Children of Korlis] to regain the life paid to :card[Griselbrand] and enable us to frequently draw our entire deck.
+
+:card[Doomsday] aims to win the game by casting the namesake card and building a pile of 5 cards to win the game, typically using :card[Thassa’s Oracle].
+
+The Tin Fins gameplan is incredibly strong in game 1, but is vulnerable to many common sideboard hate cards (:card[Leyline of the Void], :card[Endurance], :card[Surgical Extraction], and more). To work around this, the deck is able to transform into Doomsday in sideboarded games, functionally invalidating the opponents hate cards. Using :card[Doomsday] as your transformational plan works exceptionally well, as the core combo package of both plans is ~13 cards, while the shell is virtually identical for both decks.
```
[the remark this answers is not in the document]
Comment (drynne0) [reply markdown/chapters/entombsday/basics.md]: Updated as suggested

File: markdown/chapters/entombsday/additional-resources.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+---
```
[the remark this answers is not in the document]
Comment (drynne0) [reply markdown/chapters/entombsday/additional-resources.md]: Combined as suggested

File: markdown/chapters/entombsday/interaction.md (hunk 3 above)
Comment (drynne0) [reply markdown/chapters/entombsday/interaction.md]: Removed Extirpate

File: markdown/chapters/entombsday/playing-the-deck.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+## Preamble
+
+As a deck with two unique gameplans, Entombsday requires some flexibility from its pilot. Knowing the ins-and-outs of both Tin Fins and Doomsday will improve your performance with the deck, as will developing your own heuristics for when to uses which plan.
+
+Because this Wiki already contains excellent resources for how to play Turbo Doomsday, we will primarily focus on the Tin Fins plan and when to *transform* into Doomsday.
+
+## Playing Tin Fins
+
+Fundamentally, Tin Fins is an A + B reanimator combo deck.
+
```
[the remark this answers is not in the document]
Comment (drynne0) [reply markdown/chapters/entombsday/playing-the-deck.md:42]: Rewrote and expanded this section substantially

File: markdown/chapters/entombsday/playing-the-deck.md (hunk 6 above)
Comment (drynne0) [reply markdown/chapters/entombsday/playing-the-deck.md]: Added these under new "general tips" section

File: markdown/chapters/entombsday/playing-the-deck.md (hunk 5 above)
Comment (drynne0) [reply markdown/chapters/entombsday/playing-the-deck.md]: Rewrote and expanded this section substantially

File: markdown/chapters/entombsday/deck-construction.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+order: 2
+title: Deck Construction
+---
+
+## Table of Contents
+
+## Preamble
+
+As previously stated, the shell of both Tin Fins and Doomsday are remarkably similar and the combo packages are roughly the same size. Because of this overlap, there are very few concessions made in deckbuilding.
+
```
Comment (Doishy) [inline markdown/chapters/entombsday/deck-construction.md:12]: Maybe introduce the two lists and give some discussion on them and the choices made for each / reasoning for any differences?

"Here are some example decklists". You can see the shell provides the basis of Tin Fins with xyz differences. etc etc

File: markdown/chapters/entombsday/playing-the-deck.md [hunk 11]
```diff
@@ -0,0 +1,247 @@
...
+	* 	 Cast Children of Korlis to regain the life paid this turn and continue drawing cards until you can 
+
+### General Tips
+
+*  Goryo's Venegance can only target a legendary craeture. It is better to use that to reanimate your Griselbrand and save your Shallow Graves for Children of Korlis.
+*  Shallow Grave and Goryo's Vengeance create a delayed trigger that will exile the creature reanimated with them at the beginning of the next End Step. Because of this, you can cast them on your opponent's End Step and still have the creature in play for your turn.
+*  You can sacrifice the in-play Griselbrand to :card[Cabal Therapy]'s Flashback cost to put it back in your graveyard for next turn, bypassing the delayed trigger from Goryo's Vengeance/Shallow Grave.
+*  The life gain from Children of Korlis cares about total life lost this turn. In paper, make sure you track every point of life lost (fetchland activations, Thoughtseizes cast, etc.)
+*  Be mindful of both yur life total and the number of cards in your library. You don't want to draw twice from 14 life or draw seven cards with only six left in your deck.
```
Comment (Doishy) [inline markdown/chapters/entombsday/playing-the-deck.md:36]: typo - yur

File: markdown/chapters/entombsday/playing-the-deck.md [hunk 12]
```diff
@@ -0,0 +1,247 @@
...
+*  If you find that you're missing one combo piece before going to combat, you can try to cast some of your cantrips to dif a little deeper into your deck. This is also useful if you have less than seven cards left in deck and need to find a specific piece.
```
Comment (Doishy) [inline markdown/chapters/entombsday/playing-the-deck.md:37]: typo - dif

**Review by Doishy — COMMENTED** [review]
Looking better. I still think there is a lot of assumed knowledge here but deffo better. I would advocate getting a total Tin Fins/DD noob to have a read through and check it.

**Pull request merged**